src/forms/TypeaheadItem.jsx

```diff
@@ -13,6 +13,9 @@ TypeaheadItem.propTypes = {
 		PropTypes.array,
 		PropTypes.object
 	]).isRequired,
-	className: PropTypes.string,
-	children: PropTypes.node.isRequired
+	className: PropTypes.string
+	// children: PropTypes.oneOfType([
+	// 	PropTypes.node,
+	// 	PropTypes.function
+	// ]).isRequired
```

Comment:
Not sure why this is commented out. Seems like it shouldn't be?

Reply:
Yes, I need to add it back in :x